resources/install.rb

```diff
@@ -22,9 +22,9 @@
 property :source_target_arch, [String, nil], default: nil
 property :source_target_os, String, default: lazy {
   if node['kernel']['release'].split('.')[0..1].join('.').to_f > 2.6
-    @target_os = 'linux2628'
+    @target_os = 'linux-glibc'
```

Comment:
Perhaps we should only set this if HAProxy version is >= 2 as there appears to be an issue building version 1.8 and 1.9